pkgs/package-overrides.nix

```diff
@@ -579,7 +627,7 @@ in
           ];
         }
       else
-        null;
+        throw "php.extensions.xmlrpc requires PHP version < 8.0.";
```

Comment:
For extensions that were deprecated/removed, I would use “is no longer available” rather than “requires”

Reply:
I don't know the list of deprecated/removed extensions, so I used a generic message. Maybe we can refine that at some point if needed?